### ISSUE TITLE
Update attribute of enum model in migration

### ISSUE THREAD

**Issue by greyblake** [albertosaurus/power_enum#14]
Sometimes it's necessary to update an attribute of enumerated model in migration, but it's not possible since instance variable `@attributes` of model is frozen hash.

To avoid it we have to do hacks like this:

``` ruby
# duplicate forzen hash
new_attributes = model.instance_variable_get('@attributes').dup

# Assign duplication back, no we can update model attributes
model.instance_variable_set('@attributes', new_attributes)

# update attribute
model.update_attribute("attr_name", "value")

 # Freeze @attributes variable as it was before
new_attributes.freeze
```

It would be cool to have a helper or something else to handle it.

**Comment by albertosaurus** [comment]
Do you think the `update_enumerations_model` method should unfreeze the enumeration members automatically before invoking the block?

**Comment by greyblake** [comment]
Yes, good idea. It would be very convenient.

**Comment by albertosaurus** [comment]
This is resolved in version 0.9.4.
